website/docs/src/content/durable-execution.mdx

```diff
@@ -0,0 +1,326 @@
+---
+title: Durable Execution
+description: Build resilient workflows with pause/resume, retries, and human-in-the-loop capabilities
+---
+
+# Durable Execution
+
+GenSX provides durable execution capabilities that make your workflows resilient and allow for complex orchestration patterns. This includes state preservation, checkpoint restoration, and seamless human-in-the-loop integration.
+
+## Key capabilities
+
+### State preservation
+
+All workflow state is automatically preserved across restarts and failures:
```

Comment:
GenSX automatically snapshots all workflow state between component steps. If the process crashes or restarts, it picks up right where it left off: